### PR TITLE
Templates API: fix default version provider for v0.0.1

### PR DESCRIPTION
**Pull request by webrouse**
Jira: https://keboola.atlassian.net/browse/KAC-123

### CODE REVIEW

File: internal/pkg/template/repository/manifest/records.go [hunk 1]
```diff
@@ -103,7 +103,8 @@ func (v *TemplateRecord) DefaultVersion() (VersionRecord, bool) {
 	latest := VersionRecord{Version: model.ZeroSemVersion()}
 	latestStable := VersionRecord{Version: model.ZeroSemVersion()}
 	for _, item := range v.Versions {
-		if item.Version.GreaterThan(latest.Version.Value()) {
+		// GreaterThanOrEqual
+		if !item.Version.LessThan(latest.Version.Value()) {
```
Comment (webrouse) [inline internal/pkg/template/repository/manifest/records.go:107]: Monika pouziva v sablonach verziu `v0.0.1` a `ZeroSemVersion` vrati tiez `v0.0.1`, a teda podmienka je false.
Opravil som to.

File: internal/pkg/template/repository/manifest/records_test.go [hunk 2]
```diff
@@ -45,6 +45,27 @@ func TestTemplateRecord_LatestVersion_Found(t *testing.T) {
...
+
+	v, found := template.DefaultVersion()
+	assert.True(t, found)
+	assert.Equal(t, VersionRecord{
+		Version:     version(`0.0.1`),
+		Stable:      false,
+		Description: `Version 0`,
+	}, v)
+}
+
```
Comment (webrouse) [inline internal/pkg/template/repository/manifest/records_test.go:68]: A tu je test.

Teda, mechanizmus bol spravny, ale neriesil tento edge case (ktory je vlastne ale bo vsetkych sablonach od Moniky).

**Comment by webrouse** [comment]
image

**Review by JakubMatejka — APPROVED** [review]
